src/org/usfirst/frc4904/robot/commands/IdleMotors.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc4904.robot.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.usfirst.frc4904.standard.commands.motor.MotorIdle;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+/**
+ * Idles motors of a subsystem.
+ * Used as a default command
+ *
+ */
+public class IdleMotors extends CommandGroup{
```

Comment:
Also, shouldn't this belong on `standard`? I believe you and @oshlern were working on it...

Reply:
It should, but for the time being, I think its fine here